patches/chrome-browser-net-errorpage_browsertest.cc.patch

```diff
@@ -0,0 +1,94 @@
+diff --git a/chrome/browser/net/errorpage_browsertest.cc b/chrome/browser/net/errorpage_browsertest.cc
+index f157f93ae9bb7969bf9e982eddb21d2504d6d5fe..99325e07f8faef0b5456ec3429fc6e1edfd85333 100644
+--- a/chrome/browser/net/errorpage_browsertest.cc
++++ b/chrome/browser/net/errorpage_browsertest.cc
+@@ -201,6 +201,17 @@ std::string GetShowSavedButtonLabel() {
+   return l10n_util::GetStringUTF8(IDS_ERRORPAGES_BUTTON_SHOW_SAVED_COPY);
+ }
+ 
++// Returns true if the platform has support for a diagnostics tool, and it
```

Comment:
we're not running these tests right now anyway so we don't want to patch them. Can we add these to our own tests instead?

Reply:
@bridiver these are here because it's part of a cherry-pick of a chrome commit. see the commit message.